### ISSUE TITLE
prometheus.io/path should be /stats/prometheus 

### ISSUE THREAD

**Issue by alexh-sauce** [heptio/contour#976]
**What steps did you take and what happened:**

My prometheus service was complaining about failing to parse the stats, and looking at the `/stats` endpoint, it doesn't look like prometheus format.

**What did you expect to happen:**

Prometheus is able to scrape the data.

**Anything else you would like to add:**

I edited the deployment: `prometheus.io/path: "/stats/prometheus"` and it started working.

**Environment:**

- Contour version: 0.10.0
- Kubernetes version: (use `kubectl version`):

```
Client Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.3", GitCommit:"721bfa751924da8d1680787490c54b9179b1fed0", GitTreeState:"clean", BuildDate:"2019-02-04T04:48:03Z", GoVersion:"go1.11.5", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"11+", GitVersion:"v1.11.7-gke.12", GitCommit:"06f08e60069231bd21bdf673cf0595aac80b99f6", GitTreeState:"clean", BuildDate:"2019-02-25T20:37:10Z", GoVersion:"go1.10.8b4", Compiler:"gc", Platform:"linux/amd64"}
```

- Cloud provider or hardware configuration: GKE

**Comment by stevesloka** [comment]
@alexh-sauce the docs recently got updated (https://github.com/heptio/contour/blob/master/docs/prometheus.md#envoy-metrics) to match this. Previously we were using `/stats?format=prometheus`. 

There's also a PR in flight to have full prometheus / grafana examples: https://github.com/heptio/contour/pull/970

Is there an example somewhere that's incorrect?

**Comment by alexh-sauce** [comment]
@stevesloka 

I ran into this doing the example install from the README: `$ kubectl apply -f https://j.hept.io/contour-deployment-rbac`

It looks like this line will fix the issue for me:
https://github.com/heptio/contour/pull/970/files#diff-3111b29c9f100af58442a17249c9cac0R22

Although it shows up in other places as well: 

```
$ git grep prometheus.io/path
deployment/deployment-grpc-v2/02-contour.yaml:        prometheus.io/path: "/stats"
deployment/ds-grpc-v2/02-contour.yaml:        prometheus.io/path: "/stats"
deployment/ds-hostnet-split/03-envoy.yaml:        prometheus.io/path: "/stats"
deployment/ds-hostnet/02-contour.yaml:        prometheus.io/path: "/stats"
deployment/render/daemonset-rbac.yaml:        prometheus.io/path: "/stats"
deployment/render/deployment-rbac.yaml:        prometheus.io/path: "/stats"
```

**Comment by davecheney** [comment]
Can this be closed as a duplicate of #970 ?

**Comment by stevesloka** [comment]
Yes @davecheney this is a resolved in #970.
